src/RetroAchievements.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RetroAchievements\Api;
+
+use GuzzleHttp\Client;
+
+class RetroAchievements
+{
+    use MakesHttpRequests;
+
+    const API_URL = 'https://retroachievements.org/API/';
```

Comment:
I've changed this to `/API/` rather than lowercase, as otherwise it hits a 404 for all the requests. 🤷🏻

It's kind of a shame, but I assume this will be resolved when migrating to proper Laravel routes and controllers for the API. 👍🏻 Speaking of, are there any plans / a roadmap for this?